1-3/main.js

```diff
@@ -0,0 +1,26 @@
+const tasks = [
+  '掃除',
+  '買い物',
+  '散歩',
+];
+
+function tasksShow () {
```

Comment:
関数名は動詞＋名詞で命名するのが通例ですので、この場合は以下のようにしましょう
```suggestion
function showTasks () {
```